src/backend/mi2/mi2.ts

```diff
@@ -314,6 +314,20 @@ export class MI2 extends EventEmitter implements IBackend {
         });
     }
 
+    public goto(filename: string, line: number): Thenable<boolean> {
+		if (trace) {
+			this.log('stderr', 'goto');
+        }
+		return new Promise((resolve, reject) => {
+			const target: string = '"' + (filename ? escape(filename) + ":" : "") + line + '"';
```

Comment:
Two issues here
1. escape() is marked candidate for being [deprecated](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/escape)
2. Why are we escaping it that way? `escape('C:/abc d')` ==> `'C%3A/abc%20d'`. Please let us know with some proof because we can use this elsewhere.

Reply:
Hi @haneefdm, that's not the `escape()` being used. The definition is at the top of the file: https://github.com/Marus/cortex-debug/blob/631fb3f6eaaf0747686e7539a97b12c0340c2888/src/backend/mi2/mi2.ts#L12-L14

Reply:
Sorry about not seeing that. I will put your contribution into my branch and then release it. I have a ton of changes coming and it will be easier for me that way.

Reply:
Hey, no worries! Cool to hear about upcoming changes!